### PR TITLE
Weather Block Marker, Rad Markers and Invisible Walls

### PR DESCRIPTION
**Pull request by Peptide90**
Ported from Nuclear14 by request of @OldDanceJacket as they wanted the weather blocking markers. Tought you may as well get the rest of the stuff for rad markers and invisible walls too. Useful for planets.

### CODE REVIEW

File: Resources/Prototypes/Entities/Markers/environmental.yml [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+  id: MarkerRadiation1
+  parent: MarkerBase
+  suffix: intensity 1
+  components:
+  - type: Sprite
+    layers:
+      - sprite: _Nuclear14/Markers/environment.rsi
+        state: base-green
+        shader: unshaded
+      - sprite: _Nuclear14/Markers/environment.rsi
```
Comment (VMSolidus) [inline Resources/Prototypes/Entities/Markers/environmental.yml:13]:
```suggestion
      - sprite: Markers/environment.rsi
        state: base-green
        shader: unshaded
      - sprite: Markers/environment.rsi
```

File: Resources/Prototypes/Entities/Markers/environmental.yml [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+  id: MarkerBlocker
+  parent: MarkerBase
+  suffix: invisible wall
+  components:
+  - type: Sprite
+    layers:
+      - sprite: _Nuclear14/Markers/environment.rsi
+        state: base-blue
+        shader: unshaded
+      - sprite: _Nuclear14/Markers/environment.rsi
```
Comment (VMSolidus) [inline Resources/Prototypes/Entities/Markers/environmental.yml:71]:
```suggestion
      - sprite: Markers/environment.rsi
        state: base-blue
        shader: unshaded
      - sprite: Markers/environment.rsi
```

File: Resources/Prototypes/Entities/Markers/environmental.yml [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+
+# Weather Blocker
+- type: entity
+  name: Marker Weather Blocker
+  id: MarkerWeatherblocker
+  parent: MarkerBase
+  components:
+  - type: Sprite
+    sprite: _Nuclear14/Markers/environment.rsi
```
Comment (VMSolidus) [inline Resources/Prototypes/Entities/Markers/environmental.yml:98]:
```suggestion
    sprite: Markers/environment.rsi
```

**Review by VMSolidus — CHANGES_REQUESTED** [review]

**Comment by Peptide90** [comment]
My bad for copy pasting

**Review by VMSolidus — APPROVED** [review]

File: Resources/Prototypes/Entities/Markers/environmental.yml [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+# Weather Blocker
+- type: entity
+  name: Marker Weather Blocker
+  id: MarkerWeatherblocker
+  parent: MarkerBase
+  components:
+  - type: Sprite
+    sprite: Markers/environment.rsi
+    state: weather
+  - type: BlockWeather
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/Entities/Markers/environmental.yml:100]:
```suggestion
  - type: BlockWeather

```

File: Resources/Prototypes/Entities/Markers/environmental.yml [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+        shader: unshaded
+        state: wall
+  - type: PlacementReplacement
+    key: blocker
+  - type: Fixtures
+    fixtures:
+      fix1:
+        shape:
+          !type:PhysShapeAabb
+          bounds: "-0.5,-0.5,0.5,0.5"
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/Entities/Markers/environmental.yml:81]: Isn't that upside down? Probably doesn't matter much.
```suggestion
          bounds: "-0.5,0.5,-0.5,0.5"
```

**Review by DEATHB4DEFEAT — APPROVED** [review]

**Review by VMSolidus — APPROVED** [review]

**Comment by VMSolidus** [comment]
image

HOW THE FUCK DID THIS BREAK THE PR?

**Comment by Peptide90** [comment]
> image
> 
> HOW THE FUCK DID THIS BREAK THE PR?

Odd that it is literally identical to N14 but it works there no errors.

**Pull request merged**